besu/src/test/java/org/hyperledger/besu/cli/options/stable/DataStorageOptionsTest.java

```diff
@@ -34,17 +34,17 @@ public void bonsaiTrieLogPruningLimitOption() {
         dataStorageConfiguration ->
             assertThat(dataStorageConfiguration.getUnstable().getBonsaiTrieLogPruningLimit())
                 .isEqualTo(1),
-        "--Xbonsai-trie-log-pruning-enabled",
-        "--Xbonsai-trie-log-pruning-limit",
+        "--Xbonsai-limit-trie-logs-enabled",
+        "--Xbonsai-trie-logs-pruning-limit",
         "1");
   }
 
   @Test
   public void bonsaiTrieLogPruningLimitShouldBePositive() {
     internalTestFailure(
         "--Xbonsai-trie-log-pruning-limit=0 must be greater than 0",
```

Comment:
```suggestion
        "--Xbonsai-trie-logs-pruning-limit=0 must be greater than 0",
```

Reply:
There's a few other Strings where the rename needs to happen too.

Reply:
will do this in a followup PR

Reply:
Should be covered by https://github.com/hyperledger/besu/pull/6445